test/e2e/framework/util.go

```diff
@@ -3760,6 +3760,12 @@ func RunHostCmd(ns, name, cmd string) (string, error) {
 	return RunKubectl("exec", fmt.Sprintf("--namespace=%v", ns), name, "--", "/bin/sh", "-c", cmd)
 }
 
+// RunHostCmdOnContainer runs the given cmd in the context of the given pod/container
+// using `kubectl exec` inside of a shell.
+func RunHostCmdOnContainer(ns, podName, containerName, cmd string) (string, error) {
+	return RunKubectl("exec", fmt.Sprintf("--namespace=%v", ns), podName, fmt.Sprintf("--container=%v", containerName), "--", "/bin/sh", "-c", cmd)
+}
+
```

Comment:
argh. that was leftover from configmap work, thought for some reason it was being used in daemonset. will remove.